src/psyclone/tests/psyclone_test_utils.py

```diff
@@ -163,6 +180,24 @@ def get_infrastructure_flags(self):
         '''
         return []
 
+    @staticmethod
+    def skip_if_compilation_disabled():
+        '''This function is used in all tests that should only run
+        if compilation is enabled. It calls pytest.skip if compilation
+        is not enabled.'''
+        if not Compile.TEST_COMPILE:
+            return pytest.skip("Need --compile option to run")
```

Comment:
pylint points out that this probably shouldn't be a `return`?

Reply:
Correct indeed - now here start the pylint saga, since my pylint did not complain about this, till I updated pylint :)